src/pages/my-offers/MyOffers.tsx

```diff
@@ -56,6 +64,10 @@ const MyOffers = () => {
 
   const handleOpenDrawer = () => openDrawer()
 
```

Comment:
add brackets for `if` block for readabitily, unless we have a convention to omit them if the `if` block has only one statement